notebooks/README.md

```diff
@@ -11,6 +11,7 @@ Notebook | Description
 [`analyzing_income_distribution.ipynb`](https://colab.research.google.com/drive/1uZtHeQ5FJoKPdjYjaHnIPXcAe0nKLcKO)   | A notebook that plots the distribution of income using statistics provided by the 2017 [American Community Survey](https://www.census.gov/programs-surveys/acs). The final result is a histogram charting the number of individuals in income brackets ranging from "0 to 10,000USD" up to "Above 200,000USD".
 [`analyzing_obesity_prevalence.ipynb`](https://colab.research.google.com/drive/1cawpFQzuoRcZX0H_kpbBvhBNZzGjBN8t)    | A notebook that analyzes the relationship between prevalence of obesity in 500 US Cities (as provided by the [CDC Wonder](https://wonder.cdc.gov/) dataset) to health and socio-economic indicators such as prevalence of high blood pressure and poverty rate.
 [`analyzing_genomic_data.ipynb`](https://colab.research.google.com/drive/1Io7EDr4LjfPLl_l2JYY8__WbfitfNlOf) | A notebook that analyzes genetic variants within RUNX1 (provided by multiple datasets from UCSC Genome Browser, NCBI/gene, and ClinVar).
+[`Drug_Discovery_With_Data_Commons.ipynb`](https://colab.research.google.com/drive/1dSKYiRMn3mbDsInorQzYM0yk7sqv6fIV) | A notebook that explores drug repurposing, the reuse of previously approved drugs for novel applications, using Biomedical Data Commons.
```

Comment:
I think Sam had a comment about pharma companies disliking the term of repurposing so she wanted to replace everything with drug discovery

Reply:
I committed a change